play-json/shared/src/main/scala/JsPath.scala

```diff
@@ -146,11 +146,11 @@ object JsPath extends JsPath(List.empty) {
             case _ => throw new RuntimeException("when empty JsPath, expecting JsObject")
           }
           case List(p) => p match {
-            case KeyPathNode(key) => Json.obj(key -> value)
+            case KeyPathNode(key) => JsObject(Seq(key -> value))
```

Comment:
Directly use the `JsObject` constructor, rather than the factory (depending on the conversions which are useless in these cases)